styles.js

```diff
@@ -5,10 +5,24 @@ export default createGlobalStyle`
   *::before,
   *::after {
     box-sizing: border-box;
+    margin: 0;
+    padding: 0;
   }
 
   body {
-    margin: 0;
     font-family: system-ui;
+    margin: 1rem 0rem 0rem 1rem;
```

Comment:
I'm not really sure why you're "indenting" the body by 1rem top and left, but maybe there's a better way to layout your content?